simulation/__main__.py

```diff
@@ -61,6 +61,13 @@ def restrict_between_0_and_1(input_value: float):
         type=str,
     )
 
+    parser.add_argument(
+        "--override-heat-pump-suitability",
+        default=True,
```

Comment:
I think `False` is a more sensible default?
```suggestion
        default=False,
```